docs/reference/cli/subcommands.md

```diff
@@ -2320,7 +2320,7 @@ Possible values are:
 | :--------- | :--------------- | :--------- | :----------------------------------------------- |
 | `mainnet`  | Consensus layer  | Production | Ethereum main network.                           |
 | `minimal`  | Consensus layer  | Test       | Used for local testing and development networks. |
-| `goerli`   | Consensus layer  | Test       | Multi-client testnet.                            |
+| `sepolia`   | Consensus layer  | Test       | Multi-client testnet.                            |
```

Comment:
I wonder if we can link off to the docs that mentions it is permissioned.. or maybe add a note telling users it is 

https://docs.teku.consensys.io/get-started/connect/testnet

Reply:
good idea-- can't link to tab in teku, will go direct to EF